### PR TITLE
removing un-used fields

### PR DESCRIPTION
**Pull request by 311chaos**
- removing URL and method attributes and associated tests.
- adding in a loading indicator

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -19,10 +19,11 @@
...
   ],
   "dependencies": {
-    "polymer": "Polymer/polymer#^1.0.0",
-    "iron-input": "PolymerElements/iron-input#~1.0.4",
     "iron-form": "PolymerElements/iron-form#~1.0.7",
-    "ll-theme": "LeisureLinkElements/ll-theme"
+    "iron-input": "PolymerElements/iron-input#~1.0.4",
+    "ll-theme": "LeisureLinkElements/ll-theme",
+    "ll-throbber": "LeisureLinkElements/ll-throbber#fix_restructure_and_positioning",
+    "polymer": "Polymer/polymer#^1.0.0"
```
Comment (TylerGarlick) [inline bower.json]: 1.1.0?

**Comment by TylerGarlick** [comment]
:+1:

File: bower.json (hunk 1 above)
Comment (311chaos) [inline bower.json]: Ya, its the coolest.

File: bower.json [hunk 2]
```diff
@@ -19,10 +19,11 @@
     "/demo/"
   ],
   "dependencies": {
-    "polymer": "Polymer/polymer#^1.0.0",
-    "iron-input": "PolymerElements/iron-input#~1.0.4",
     "iron-form": "PolymerElements/iron-form#~1.0.7",
-    "ll-theme": "LeisureLinkElements/ll-theme"
+    "iron-input": "PolymerElements/iron-input#~1.0.4",
+    "ll-theme": "LeisureLinkElements/ll-theme",
+    "ll-throbber": "LeisureLinkElements/ll-throbber#fix_restructure_and_positioning",
```
Comment (zack37) [inline bower.json]: Does this need to be changed back to master?

Comment (311chaos) [inline bower.json]: yar, was waiting for its approval.

**Comment by zack37** [comment]
:+1:

**Pull request merged**